tools/update_repo.sh

```diff
@@ -12,8 +12,6 @@ output_paths=(
     "dev_envs/default_python3/Pipfile"
     "dev_envs/default_python3/Pipfile.lock"
     "dev_envs/pytest/conftest.py"
-    "dev-requirements-py2.txt"
-    "dev-requirements-py3.txt"
```

Comment:
@ilaner please have a look at this file, we may need to update it to support poetry too (add pyproject, lock, etc)